templates/microprofile-kumuluzee/files/.platform.template.yaml

```diff
@@ -0,0 +1,53 @@
+# Platform.sh Project Initialization Template
+#
+# This file defines settings and workflow modifications that allow a git
+# repository to be deployed to Platform.sh and its white-label partners. A
+# project template can be a fully functioning ready-made application or a
+# quick-start point for custom development work.
+#
+# It contains elements that affect the behaviour upon the initialisation of
+# a new project (for example minimal plan sizes) as well as elements that
+# allow Platform.sh to present it in a user interface (such as the description
+# of the project, tags, an icon etc.).
+#
+# Once provisioned this file has no effects on the running project.  You
+# are free to remove it.
+
+# The schema is versioned so that we can establish code paths differently in the future if we need to change this.
+version: 1
+
+# Templates are a small amount of information supporting a template URL. Each template is selectable at the project-creation step.
+info:
+  # Unique machine name, prefaced by a vendor or organization identifier
+  id: platformsh/kumuluzee-microprofile
+  # The human-readable name of the template.
+  name: Eclipse MicroProfile, KumuluzEE, Maven
+  # Human-readable descriptive text for the template. Supports limited HTML.
+  description: Eclipse MicroProfile by KumuluzEE
+  # A list of tags associated with the template.
+  tags:
+  - Java
```

Comment:
I remember we came across this before: these are tags for organizing templates in the UI. After Java it's things like "Framework" and "CMS"